src/main/java/org/apache/commons/imaging/formats/gif/GifImageParser.java

```diff
@@ -485,24 +495,35 @@ public Dimension getImageSize(final ByteSource byteSource, final Map<String, Obj
         if (bhi == null) {
             throw new ImageReadException("GIF: Couldn't read Header");
         }
-
-        final ImageDescriptor id = (ImageDescriptor) findBlock(blocks.blocks,
-                IMAGE_SEPARATOR);
-        if (id == null) {
-            throw new ImageReadException("GIF: Couldn't read ImageDescriptor");
-        }
-
-        // Prefer the size information in the ImageDescriptor; it is more
-        // reliable
-        // than the size information in the header.
-        return new Dimension(id.imageWidth, id.imageHeight);
+        
```

Comment:
This looks fine for me, but Travis appears to be complaining about this line? :confused: 

```
[ERROR] src/main/java/org/apache/commons/imaging/formats/gif/GifImageParser.java:[498] (regexp) RegexpSingleline: Line has trailing spaces.
```

Reply:
Thanks, I saw. Had some trailing spaces. Fixed.